experiments/tester/src/RNTester/TestComponents/RadioGroupTest.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { RadioButton, RadioGroup } from 'react-native-uifabric';
+// import { RadioButton } from '../../components';
+// import { RadioGroup } from '../../components';
+
+export const RadioGroupTest: React.FunctionComponent<{}> = () => {
+  // Client's example onChange function
+  const onChange = (key: string) => {
```

Comment:
You should create a proper function here so that React doesn't re-render if not necessary.

Reply:
use React.useCallback so you can specify when to re-render

Reply:
`useCallback`?

Reply:
Something like this? 
  const onChange = React.useCallback((key: string) => {
    return;
  }, []);

Reply:
If the callback never changes, creating a normal function is the simplest solution. No need to call `useCallback`:

```
function onChange(key: string): void {
}
```

Reply:
Changed to the above